src/store/Branding.ts

```diff
@@ -13,6 +13,7 @@
 import { Action, Reducer } from 'redux';
 import { fetchBranding } from '../services/assets/branding';
 import { AppThunkAction, AppState } from '.';
+import { merge } from 'lodash';
```

Comment:
Have you added `lodash` into `product.json`?

Reply:
I have added it into ```package.json```.